native/src/scylladb_connection.cpp

```diff
@@ -0,0 +1,646 @@
+/**
+ * ScyllaDB Migrator - ScyllaDB Connection Implementation
+ * 
+ * Uses cpp-rs-driver (0.5.1) for ScyllaDB operations
+ * 
+ * Copyright (c) 2025 ScyllaDB
+ * Licensed under Apache License 2.0
+ */
+
+#include "mariadb_scylla_migrator.h"
+
+#include <sstream>
+#include <algorithm>
+#include <chrono>
+#include <thread>
+#include <cstring>
+
+namespace scylla_migrator {
+namespace mariadb {
+
+// ============================================================================
+// ScyllaDBConnection Implementation
+// ============================================================================
+
+ScyllaDBConnection::ScyllaDBConnection(const ScyllaDBConfig& config)
```

Comment:
eh ?

Scylla is supported in spark, we don't need a cpp connection to it
not mentioning cpp driver for scylla is obsolete and it will be replaced by its cpp-over-rust variant

so this code is completely useless and unoptimal